### PR TITLE
PKG_CONFIG_PATH search extended to {lib,lib64,lib32,share}/pkgconfig

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/framework/easyblock.py [hunk 1]
```diff
@@ -1065,7 +1065,7 @@ def make_module_req_guess(self):
             'LIBRARY_PATH': ['lib', 'lib64', 'lib32'],
             'CPATH': ['include'],
             'MANPATH': ['man', 'share/man'],
-            'PKG_CONFIG_PATH': ['lib/pkgconfig', 'share/pkgconfig'],
+            'PKG_CONFIG_PATH': ['%s/pkgconfig' % x for x in ['lib', 'lib64', 'lib32', 'share']],
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: I'd prefer this (use `os.path.join` + alphabetical order):

```python
    'PKG_CONFIG_PATH': [os.path.join(x, 'pkgconfig') for x in ['lib', 'lib32', 'lib64', 'share']],
```

Comment (hajgato) [inline easybuild/framework/easyblock.py]: @boegel Why alphabetic? then should not be `'LIBRARY_PATH': ['lib', 'lib64', 'lib32']` also alphabetic?
And os.path.join for manpaht as well?

Comment (boegel) [inline easybuild/framework/easyblock.py]: @hajgato: style, mostly, the order should be changed for other stuff too, indeed

for `'share/man'`, it's a single string, not a composed string, but it doesn't hurt to also use `os.path.join` there

**Comment by hajgato** [comment]
@boegel done

**Comment by boegel** [comment]
@hajgato: thanks, look good, will merge when Jenkins gives the green light

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**